plugin/pkg/admission/limitranger/admission_test.go

```diff
@@ -219,15 +219,44 @@ func TestMergePodResourceRequirements(t *testing.T) {
 	}
 	verifyAnnotation(t, &pod, "LimitRanger plugin set: cpu, memory request for container foo-0; cpu, memory limit for container foo-0")
 
-	// pod with some resources enumerated should only merge empty
+	// pod with some resources enumerated should only merge empty, only default cpu
 	input := getResourceRequirements(getComputeResourceList("", "512Mi"), getComputeResourceList("", ""))
-	pod = validPodInit(validPod("limit-memory", 1, input), input)
+	pod = validPodInit(validPod("limit-cpu", 1, input), input)
```

Comment:
I would expect us to preserve the old test, update it as necessary, and add a new case for any changed behaviour. We almost do that but not quite?

Reply:
ok